test/lit/exec/fuzzing-api.wast

```diff
@@ -118,6 +120,139 @@
    )
   )
  )
+
+ ;; CHECK:      [fuzz-exec] calling ref.calling
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 42]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 3.14159]
+ ;; CHECK-NEXT: [exception thrown: __private ()]
+ (func $ref.calling (export "ref.calling")
+  ;; This will emit some logging.
+  (call $call.ref
+   (ref.func $logging)
+  )
+  ;; This will trap.
+  (call $call.ref
+   (ref.null func)
+  )
+ )
+
+ ;; CHECK:      [fuzz-exec] calling ref.calling.catching
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 42]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 3.14159]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 0]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 1]
+ (func $ref.calling.catching (export "ref.calling.catching")
+  ;; This will emit some logging, then log 0 as we do not error.
+  (call $log-i32
+   (call $call.ref.catch
+    (ref.func $logging)
+   )
+  )
+  ;; The trap here is caught, and we'll log 1.
```

Comment:
```suggestion
  ;; The exception here is caught, and we'll log 1.
```
Because we explicitly let traps propagate, right?

Reply:
Correct, yes. Thanks, this was indeed inaccurate.